images/access-setup/content/bin/setup_compute.sh

```diff
@@ -147,6 +147,33 @@ init() {
   manifests_dir="$WORK_DIR/credentials/manifests"
   mkdir -p "$credentials_dir"
   mkdir -p "$manifests_dir"
+
+  detect_container_engine
+}
+
+detect_container_engine() {
+    CONTAINER_ENGINE="${CONTAINER_ENGINE:-}"
```

Comment:
Do you see value to actually add this as a parameter to the script? Because, currently, user will have to export 'CONTAINER_ENGINE' before running the script. I don't see this documented in a README anywhere and even if we do document it, its a possibility that they might miss it.
Same goes for the other script too.

Reply:
We use that in other scripts. What do you think about documenting that in the DEPENDENCIES.md?

Reply:
I could only find this being used in local. And its already [documented](https://github.com/openshift-pipelines/pipeline-service/blob/main/DEVELOPMENT.md?plain=1#L31) in development.md.

Given the number of people that would really need this is less, I guess we document the same under ckcp/README.md and leave it be (instead of parameterizing it).

Reply:
I've documented that bit, please close the conversation if that matches your expectations.